pkg/oc/cli/cmd/rsync/copyrsync.go

```diff
@@ -111,6 +110,21 @@ func (r *rsyncStrategy) Validate() error {
 	return nil
 }
 
+// rsyncEscapeCsommand wraps each element of the command in double quotes
+// if it contains any of the following: single quote, double quote, backslash, or space
+// It also replaces every pre-existing double quote character in the element with a pair.
+// Example: " -> ""
+func rsyncEscapeCommand(command []string) []string {
+	for key := range command {
+		needsQuoted := strings.ContainsAny(command[key], `'" \`)
+		if needsQuoted {
+			command[key] = strings.Replace(command[key], `"`, `""`, -1)
```

Comment:
I may need to brush up on my escaping, how does replacing a " with a "" help?  wouldn't we want to replace a " with a \" ?